### PR TITLE
Add Altair fork epoch for Prater.

### LINKED ISSUE #4273: Update Prater config
### Description
Include new Prater config with Altair upgrade epoch as per https://github.com/eth2-clients/eth2-networks/pull/58

### PR DESCRIPTION
**Pull request by ajsutton**
## PR Description
Add Altair for epoch for Prater. Upgrades Prater to new preset based config.

## Fixed Issue(s)
fixes #4273 

## Documentation

- [x] I thought about documentation and added the `documentation` label to this PR if updates are required.

## Changelog

- [x] I thought about adding a changelog entry, and added one if I deemed necessary.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ For information on changes in released versions of Teku, see the [releases page]
  - If you have `--Xdata-storage-non-canonical-blocks-enabled` set, this option has been renamed to `--data-storage-non-canonical-blocks-enabled`.
 
 ### Additions and Improvements
+ - Scheduled Altair upgrade on the Prater testnet at epoch 36660.
```
Comment (rolfyone) [inline CHANGELOG.md:20]: i mean, i know this is a group decision, but 666 ... really....

**Review by rolfyone — APPROVED** [review]
LGTM

**Review by jgundy1985 — CHANGES_REQUESTED** [review]
[]()

File: ethereum/spec/src/test/java/tech/pegasys/teku/spec/SpecFactoryTest.java [hunk 2]
```diff
@@ -31,7 +31,7 @@
 
 public class SpecFactoryTest {
 
-  private static final Set<String> ALTAIR_NETWORKS = Set.of("pyrmont");
+  private static final Set<String> ALTAIR_NETWORKS = Set.of("pyrmont", "prater");
```
Comment (jgundy1985) [inline ethereum/spec/src/test/java/tech/pegasys/teku/spec/SpecFactoryTest.java:34]: 34 + new account  DEPOSIT_CONTRACT_ADDRESS: 0xff50ed3d0ec03aC01D4C79aAd74928BFF48a7b2b

Comment (jgundy1985) [reply ethereum/spec/src/test/java/tech/pegasys/teku/spec/SpecFactoryTest.java:34]: - [ ] A

File: util/src/main/resources/tech/pegasys/teku/util/config/configs/prater.yaml [hunk 3]
```diff
@@ -1,156 +1,74 @@
-# Prater preset
+# Extends the mainnet preset
+PRESET_BASE: 'mainnet'
 
 # For backwards compatibility in the config/spec API
 CONFIG_NAME: "prater"
 
-# Misc
+# Prater config
+# Genesis
```
Comment (jgundy1985) [inline util/src/main/resources/tech/pegasys/teku/util/config/configs/prater.yaml:8]: - [ ] []()__

File: util/src/main/resources/tech/pegasys/teku/util/config/configs/prater.yaml [hunk 4]
```diff
@@ -1,156 +1,74 @@
...
-# Deposit contract
-# ---------------------------------------------------------------
-# Ethereum Goerli testnet
-DEPOSIT_CHAIN_ID: 5
-DEPOSIT_NETWORK_ID: 5
-# Prater test deposit contract on Goerli Testnet
-DEPOSIT_CONTRACT_ADDRESS: 0xff50ed3d0ec03aC01D4C79aAd74928BFF48a7b2b
+# Prater area code (Vienna)
+GENESIS_FORK_VERSION: 0x00001020
+# Customized for Prater: 1919188 seconds (Mar-23-2021 02:00:00 PM +UTC)
```
Comment (jgundy1985) [inline util/src/main/resources/tech/pegasys/teku/util/config/configs/prater.yaml:16]: []()

File: util/src/main/resources/tech/pegasys/teku/util/config/configs/prater.yaml [hunk 5]
```diff
@@ -1,156 +1,74 @@
...
-# 2**0 * 10**9 (= 1,000,000,000) Gwei
-MIN_DEPOSIT_AMOUNT: 1000000000
-# 2**5 * 10**9 (= 32,000,000,000) Gwei
-MAX_EFFECTIVE_BALANCE: 32000000000
-# 2**4 * 10**9 (= 16,000,000,000) Gwei
-EJECTION_BALANCE: 16000000000
-# 2**0 * 10**9 (= 1,000,000,000) Gwei
-EFFECTIVE_BALANCE_INCREMENT: 1000000000
+# Some forks are disabled for now:
+#  - These may be re-assigned to another fork-version later
```
Comment (jgundy1985) [inline util/src/main/resources/tech/pegasys/teku/util/config/configs/prater.yaml:23]:
```suggestion
#  - These may be re-assigned to another fork-version later
```

File: util/src/main/resources/tech/pegasys/teku/util/config/configs/prater.yaml [hunk 6]
```diff
@@ -1,156 +1,74 @@
...
+SHARDING_FORK_VERSION: 0x03001020
+SHARDING_FORK_EPOCH: 18446744073709551615
 
-# Initial values
-# ---------------------------------------------------------------
-# Prater area code (Vienna)
-GENESIS_FORK_VERSION: 0x00001020
-BLS_WITHDRAWAL_PREFIX: 0x00
+# TBD, 2**32 is a placeholder. Merge transition approach is in active R&D.
+TRANSITION_TOTAL_DIFFICULTY: 4294967296
```
Comment (jgundy1985) [inline util/src/main/resources/tech/pegasys/teku/util/config/configs/prater.yaml:37]:
```suggestion
TRANSITION_TOTAL_DIFFICULTY: 4294967296
```

File: util/src/main/resources/tech/pegasys/teku/util/config/configs/prater.yaml [hunk 7]
```diff
@@ -1,156 +1,74 @@
...
 
 
 # Time parameters
 # ---------------------------------------------------------------
-# Customized for Prater: 1919188 seconds (Mar-23-2021 02:00:00 PM +UTC)
-GENESIS_DELAY: 1919188
 # 12 seconds
```
Comment (jgundy1985) [inline util/src/main/resources/tech/pegasys/teku/util/config/configs/prater.yaml:42]:
```suggestion
 # 12 seconds
```

File: util/src/main/resources/tech/pegasys/teku/util/config/configs/prater.yaml [hunk 8]
```diff
@@ -1,156 +1,74 @@
...
-MIN_SEED_LOOKAHEAD: 1
-# 2**2 (= 4) epochs 25.6 minutes
-MAX_SEED_LOOKAHEAD: 4
-# 2**6 (= 64) epochs ~6.8 hours
-EPOCHS_PER_ETH1_VOTING_PERIOD: 64
-# 2**13 (= 8,192) slots ~13 hours
-SLOTS_PER_HISTORICAL_ROOT: 8192
+# 14 (estimate from Eth1 mainnet)
+SECONDS_PER_ETH1_BLOCK: 14
 # 2**8 (= 256) epochs ~27 hours
```
Comment (jgundy1985) [inline util/src/main/resources/tech/pegasys/teku/util/config/configs/prater.yaml:46]:
```suggestion
 # 2**8 (= 256) epochs ~27 hours
```

File: util/src/main/resources/tech/pegasys/teku/util/config/configs/prater.yaml [hunk 9]
```diff
@@ -1,156 +1,74 @@
...
 MIN_VALIDATOR_WITHDRAWABILITY_DELAY: 256
 # 2**8 (= 256) epochs ~27 hours
```
Comment (jgundy1985) [inline util/src/main/resources/tech/pegasys/teku/util/config/configs/prater.yaml:48]:
```suggestion
 # 2**8 (= 256) epochs ~27 hours
```

File: util/src/main/resources/tech/pegasys/teku/util/config/configs/prater.yaml [hunk 10]
```diff
@@ -1,156 +1,74 @@
...
 SHARD_COMMITTEE_PERIOD: 256
```
Comment (jgundy1985) [inline util/src/main/resources/tech/pegasys/teku/util/config/configs/prater.yaml:49]:
```suggestion
 SHARD_COMMITTEE_PERIOD: 256
```

File: util/src/main/resources/tech/pegasys/teku/util/config/configs/prater.yaml [hunk 11]
```diff
@@ -1,156 +1,74 @@
...
-# 2**3 (= 8)
-PROPOSER_REWARD_QUOTIENT: 8
-# 2**26 (= 67,108,864)
-INACTIVITY_PENALTY_QUOTIENT: 67108864
-# 2**7 (= 128) (lower safety margin at Phase 0 genesis)
-MIN_SLASHING_PENALTY_QUOTIENT: 128
-# 1 (lower safety margin at Phase 0 genesis)
-PROPORTIONAL_SLASHING_MULTIPLIER: 1
+# 2**11 (= 2,048) Eth1 blocks ~8 hours
+ETH1_FOLLOW_DISTANCE: 2048
```
Comment (jgundy1985) [inline util/src/main/resources/tech/pegasys/teku/util/config/configs/prater.yaml:51]:
```suggestion
ETH1_FOLLOW_DISTANCE: 2048
```

**Review by jgundy1985 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Loveisweird) [reply CHANGELOG.md:20]: Yup

**Pull request merged**